plugins/kn-plugin-source-kafka/artifacthub-pkg.yml

```diff
@@ -0,0 +1,31 @@
+# Artifact Hub package metadata file
+# See https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 1.9.0
+name: kn-plugin-source-kafka
+displayName: Knative Client Plugin Source Kafka
```

Comment:
The question for me would be, how this display name is used. For me, "Kafka Source Plugin" would read much nicer if the context "Knative Client Plugin" is given (i.e. if the display name is only used in this context). If this appears e.g. as title of a catalog, I would think this is too bulky, also the important part ("Kafka") comes last, so in a sorted list, this is also not easy detectable.